.travis.yml

```diff
@@ -0,0 +1,49 @@
+sudo: required
+
+language: go
+go:
+    - 1.5.1
+    - release
+
+services:
+    - docker
+
+env:
+    global:
+        - DOCKER_VERSION=1.10.1-0~trusty
+        - DOCKER_COMPOSE_VERSION=1.6.0
+
+before_install:
+    # list docker-engine versions
+    - apt-cache madison docker-engine
+
+    # upgrade docker-engine to specific version
+    - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine=${DOCKER_VERSION}
+
+    # reinstall docker-compose at specific version
+    - sudo rm /usr/local/bin/docker-compose
+    - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+    - chmod +x docker-compose
+    - sudo mv docker-compose /usr/local/bin
+    - docker-compose build
+
+go_import_path: github.com/cfpb/rhobot
+
+install:
+  - go get github.com/Sirupsen/logrus
+  - go get github.com/ahl5esoft/golang-underscore
+  - go get github.com/codegangsta/cli
+  - go get -u github.com/davecgh/go-spew/spew
+  - go get github.com/lib/pq
+  - go get gopkg.in/yaml.v2
+  - go get github.com/flosch/pongo2
+  - go get gopkg.in/gomail.v2
+
+script:
+    # setup dockerized services
+    #- docker version
```

Comment:
the two version commands have been removed. They were leftovers from the original travis file I found and pilfered.